packages/expo-constants/ios/EXConstants.podspec

```diff
@@ -22,4 +22,17 @@ Pod::Spec.new do |s|
   else
     s.source_files = "#{s.name}/**/*.{h,m}"
   end
+
+  s.script_phase = {
+    :name => 'Generate app.config for prebuilt Constants.manifest',
+    :script => File.join(__dir__, '..', 'scripts', 'get-app-config-ios.sh'),
+    :execution_position => :before_compile
+  }
```

Comment:
Imho this is not acceptable solution because users running `pod install` will get a warning
```
[!] EXConstants has added 1 script phase. Please inspect before executing a build. See `https://guides.cocoapods.org/syntax/podspec.html#script_phases` for more information.
```
which might be confusing for them and induce concerns

Reply:
personally for security reasons, i like this warning.
i will know what is going to happen at build time.
compared to npm transitive dependencies with postinstall script, it is hard for developers to find malicious code.

well, if this will be a concern, we could yet again to suppress this warning by cocoapods a hook.
```ruby
module Pod
  class Installer
    private
      define_method(:warn_for_installed_script_phases) do
        # no-op
      end
  end
end
```

let me know how you think.

Reply:
It's totally ok that CocoaPods shows this warning and I agree with the security reasons, but because of this the users may think we do something dangerous and it sheds a negative light on the package. This will need to be moved to a script that is run by autolinking during `pod install` (but I need some time to get this implemented).

Reply:
if i didn't misunderstand, `getAppConfig.js` should be executed after build but not just `pod install`.
is it possible to have this chance by incremental build?